package.json

```diff
@@ -23,13 +23,12 @@
   "homepage": "https://github.com/wikimedia/restbase-mod-table-sqlite",
   "dependencies": {
     "bluebird": "^3.5.2",
-    "cassandra-uuid": "^0.1.0",
     "extend": "^3.0.2",
     "fast-json-stable-stringify": "^2.0.0",
     "generic-pool": "^3.4.2",
     "js-yaml": "^3.12.0",
     "lru-cache": "^4.1.3",
-    "restbase-mod-table-spec": "^1.1.2",
+    "restbase-mod-table-spec": "^1.1.3",
```

Comment:
Why is this bump needed here?

Reply:
technically not needed, but 1.1.3 is removing the cass-uuid dependency from the spec, so this is removing the transitive dependency.